### PR TITLE
Increase max denylist size in the full_denylist scenarios

### PR DESCRIPTION
**Pull request by M4TTRX**
## Description

In the `APPSEC_BLOCKING_FULL_DENYLIST` scenario: 
- Increase the number of IPs blocked: 1250 ➡️  12500 (x10)
- Increase the number of users blocked: 1250 ➡️ 2500 (x2)
New total of entries blocked: 2500 ➡️ **15000**

Other changes:
- Added a block one ip and block one user test
- Only test the first and last IP blocked ip and user, to speed up the tests.
- 
## Motivation

One of our clients requires blocking of up to 15000 IPs at once. We are currently seeing if it would be possible in the next quater to increase the limit of IPs that can be blocked from 2500 IPs to 15000. This test is the first part that checks the feasibility of such increase.

## Reviewer checklist

* [ ] If this PR modify anything else than sctriclty the default scenario, then remove the `run-default-scenario` label
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

> **_NOTE:_**  By default in PR only default scenario tests will be launched. Temove `run-default-scenario` label to run all scenarios ([more info](https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2866381467/CI+Workflow+Github+Actions))

Once your PR is reviewed, you can merge it ! :heart:

### CODE REVIEW

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
Some failure on java on this scenario.

**Comment by smola** [comment]
@M4TTRX Feel free to add a `@bug(library="java", reason="RC payload limit")` to the failing test and move on. I'll modify it once we have a release with a fix.

**Review by smola — APPROVED** [review]

**Review by simon-id — APPROVED** [review]

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/appsec/test_user_blocking_full_denylist.py [hunk 1]
```diff
@@ -57,7 +74,10 @@ def setup_blocking_test(self):
         interfaces.library.wait_for(self._remote_config_asm_payload, timeout=30)
         interfaces.library.wait_for(self._remote_config_is_applied, timeout=30)
 
-        self.r_blocked_requests = [weblog.get("/users", params={"user": i}) for i in range(1250)]
+        self.r_blocked_requests = [
+            weblog.get("/users", params={"user": 1}),
```
Comment (cbeauchesne) [inline tests/appsec/test_user_blocking_full_denylist.py:78]: either 0 - 2499, or 1 - 2500

File: tests/appsec/test_blocking_addresses.py [hunk 2]
```diff
@@ -49,6 +49,23 @@
...
+    def setup_block_user(self):
+        self.block_user_req = weblog.get("/users", params={"user": "blockedUser"})
+
+    @missing_feature(library="java", reason="Missing /users endpoint")
+    def test_block_user(self):
+        """can block the request from the user"""
+
+        assert self.block_user_req.status_code == 403
+
     """Test the addresses supported for blocking"""
```
Comment (cbeauchesne) [inline tests/appsec/test_blocking_addresses.py:69]: Move up this docstring

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/appsec/test_blocking_addresses.py (hunk 2 above)
Comment (M4TTRX) [reply tests/appsec/test_blocking_addresses.py]: done

File: tests/appsec/test_user_blocking_full_denylist.py (hunk 1 above)
Comment (M4TTRX) [reply tests/appsec/test_user_blocking_full_denylist.py]: done

File: tests/appsec/test_blocking_addresses.py [hunk 3]
```diff
@@ -49,6 +49,26 @@
...
+
+    def setup_block_ip(self):
+        self.block_ip_req = weblog.get(headers={"X-Forwarded-For": "1.1.1.1"})
+
+    def test_block_ip(self):
+        """can block the request forwarded for the ip"""
+
+        assert self.block_ip_req.status_code == 403
+
+    """Test the blocking of one user"""
```
[the remark this answers is not in the document]
Comment (M4TTRX) [reply tests/appsec/test_blocking_addresses.py]: done

File: tests/appsec/test_blocking_addresses.py [hunk 4]
```diff
@@ -49,6 +49,26 @@
 @missing_feature(weblog_variant="spring-boot-3-native", reason="GraalVM. Tracing support only")
 @missing_feature(weblog_variant="akka-http", reason="Missing support")
 class Test_BlockingAddresses:
+    """Test the blocking of one IP"""
```
[the remark this answers is not in the document]
Comment (M4TTRX) [reply tests/appsec/test_blocking_addresses.py]: doen

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**